src/options/views/tab-installed.vue

```diff
@@ -63,8 +63,11 @@
           </div>
         </dropdown>
         <div class="filter-search hidden-sm">
-          <input type="search" :placeholder="i18n('labelSearchScript')" :title="searchError"
-                 v-model="search">
+          <!-- form and id is required for the built-in autocomplete using entered values -->
+          <form @submit.prevent>
```

Comment:
How about changing the outer `<div>` to `<form>` so that the DOM structure does not need to be changed?

Reply:
Sounds good.